package.json

```diff
@@ -90,8 +90,7 @@
   },
   "jupyterlab": {
     "extension": true,
-    "outputDir": "jupyterlab_filesystem_access/labextension",
-    "schemaDir": "schema"
```

Comment:
Just to double check: is this deletion related to the fix in https://github.com/jupyterlab/jupyterlab/pull/12556 now available in JupyterLab 3.4.3?

Reply:
No it's unrelated to this. The schema was used for disabling some context menus for rename/copy/cut/paste https://github.com/jupyterlab-contrib/jupyterlab-filesystem-access/blob/06f3e91d4fbb563dd18de355ca667b96aad00ad8/schema/plugin.json, it's not needed anymore since those commands are implemented

Reply:
ah right, thanks for the context :+1: